tutorials-and-examples/hf-tgi/main.tf

```diff
@@ -72,6 +72,15 @@ resource "kubernetes_deployment" "inference_deployment" {
       }
 
       spec {
+        init_container {
+          name    = "download-model"
+          image   = "google/cloud-sdk:latest"
```

Comment:
Can we pin the version instead of using latest? Also wondering how big this image is, is there a smaller image with just `gsutil`?

Reply:
`google/cloud-sdk` is ~2.8G. I switched to `google/cloud-sdk:alpine` which is around 750M